src/tritonserver.cc

```diff
@@ -1199,6 +1205,16 @@ TRITONSERVER_ServerOptionsSetBufferManagerThreadCount(
   return nullptr;  // Success
 }
 
+TRITONAPI_DECLSPEC TRITONSERVER_Error*
+TRITONSERVER_ServerOptionsSetModelLoadThreadCount(
+    TRITONSERVER_ServerOptions* options, unsigned int thread_count)
+{
+  TritonServerOptions* loptions =
+      reinterpret_cast<TritonServerOptions*>(options);
+  loptions->SetModelLoadThreadCount(thread_count);
```

Comment:
Question: Since this is a user provided variable, should we set an upper bound or is that being too opinionated? The user could potentially set thread count to -1 and lock up the system by generating a large number of threads. What would be a reasonable max here, I'm thinking `2 * std::thread::hardware_concurrency()` or `3 * std::thread::hardware_concurrency()`

Reply:
Personally, I think it's opinionated to limit it, it's up to the user to pick their desired value if they choose to set it at all. If they don't set it, they will get a reasonable default.